content/leave.rst

```diff
@@ -6,7 +6,9 @@ Leave / Holiday
 General
 =======
 
-* We have 30 days of paid holidays per year (21 legal leave + 9 bank holidays)
+* We have 32 days of paid holidays per year (21 legal leave + 11 bank holidays)
+
+* After 2 years within the team, another leave day is added to the total amount.
```

Comment:
Is this after every two years? Or just after the first two years? Please make it as clear as possible. Thanks!